src/chains/tests/test_models.py

```diff
@@ -18,6 +18,13 @@ def test_str_method_outputs_name_chain_id(self) -> None:
         )
 
 
+class ChainLogoTestCase(TestCase):
+    def test_chain_logo_upload_path(self) -> None:
+        chain = ChainFactory.create(id=69)
+
+        self.assertEqual(chain.chain_logo_uri.url, "/media/chains/69/chain_logo.jpg")
```

Comment:
You can use the generated `chain.id` (generated by Faker) to validate the path also (to cover random chain ids).

Reply:
I adjusted it in 82083887dcd88a0e34c9b70fe269a468686c90e2, along with that of the `currency_logo` for consistency.